### ISSUE TITLE
Can the example of labels guide me?

### ISSUE THREAD

**Issue by ts19970816** [HuchieWuchie/mask-rcnn-pytorch#1]

**Comment by HuchieWuchie** [comment]
Hi there. This repository was made for me to train and experiment with a Mask rcnn on google colab. I would suggest that you use some other repository. There is also this fine tutorial on how to get started with mask rcnn in pytorch. https://pytorch.org/tutorials/intermediate/torchvision_tutorial.html
